vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd

```diff
@@ -33,59 +33,59 @@ Average acceleration is known to be correlated with the activity-related energy
+The distribution of acceleration values is often referred to as intensity distribution in the physical activity literature.
 
 ## Derived descriptives
 
 ### Sets of quantiles (MX metrics by [Rowlands et al.](https://pubmed.ncbi.nlm.nih.gov/29360664/))
 
-The quantiles as discussed above can be used to describe the accelerations that participants exceed in their most active “X” accumulated minutes in a day.
-In the specific approach as proposed by [Rowlands et al.](https://pubmed.ncbi.nlm.nih.gov/31808014/) these quantiles are referred to as the **MX metrics**.
+The quantile,s as discussed above, can be used to describe the accelerations that participants exceed in their most active “X” accumulated minutes in a day.
+In the specific approach, as proposed by [Rowlands et al.](https://pubmed.ncbi.nlm.nih.gov/31808014/), these quantiles are referred to as the **MX metrics**.
 The MX metrics should not be confused with the most active continuous X hours, e.g.
 M10, as used in circadian rhythm research that also can be derived with GGIR (see parameter `winhr`).
 
-To use the MX metrics as proposed by Rowlands et al., specify the durations of the 24h day that you wish to filter out the accelerations for.
-For example, to generate the minimum acceleration value for the most active 30 minutes, you can call `qlevels = (1410/1440)`, which will filter out the lowest 1410 minutes of the day.
-This can also be used as a nested term to generate multiple metrics, for example to call M60, M30 and M10, you can use the following parameter:
+To use the MX metrics as proposed by Rowlands et al., specify the durations of the 24h day during which you want to filter out the accelerations values.
+For example, to generate the minimum acceleration value for the most active accumulated 30 minutes, you can call `qlevels = (1410/1440)`, which will filter out the lowest 1410 minutes of the day.
+This parameter also accepts nested term to generate multiple MX metrics. For example, to call M60, M30, and M10, you can specify the following:
 
 `qlevels = c(c(1380/1440), c(1410/1440), c(1430/1440))`.
 
-Note: if parameter `qwindow` has values less than 24 h, e.g. the school day (e.g. [Fairclough et al 2020](https://pubmed.ncbi.nlm.nih.gov/31593604/)), the denominator should be changed from 1440 (24h) to the new `qwindow` window length, e.g. if an 8-hour window is used then the denominator would be 480 rather than 1440 and the numerator would be 480 minus for example 60, 30, 10 or 5.
+Note: If parameter `qwindow` has values below 24 h, e.g. the school day (e.g. [Fairclough et al 2020](https://pubmed.ncbi.nlm.nih.gov/31593604/)), the denominator should be changed from 1440 (24h) to the new `qwindow` window length. If for instance an 8-hour window is used, then the denominator would be 480 rather than 1440 and the numerator would be 480 minus for example 60, 30, 10, or 5.
```

Comment:
qwindow can be used to segment the day in many different segment lengths, by which there is no single but multiple window lengths. Discussing the new window length makes this unnecessarily confusing.

Can you rephrase such that it is clear that this only works for one window length and that GGIR currently does not facilitate using a standardised duration for multiple unequal day segments.